PR: https://github.com/odoo/odoo/pull/178200

From: f10bd8ddcf99c9247dca417fefbc260454d74ea1
From: Gorash
Date: 2024-10-15 09:33:02

Breaking data model changes scores: del:4 + add:0, change matches:
-    _inherit = 'fleet.vehicle.model.category'
-    _inherit = 'stock.location'
-    _inherit = "stock.picking"
-    _inherit = 'stock.picking.batch'

Total Changes: 10

[IMP] *: adapt model class names to correspond to model names (apply script)

Part-of: odoo/odoo#178200
Related: odoo/enterprise#69762
Related: odoo/documentation#11115
Related: odoo/upgrade#6577
Related: odoo/design-themes#988
Related: odoo/upgrade-util#143

================================= pseudo patch: =================================

--- a/addons/stock_fleet/models/fleet_vehicle_model.py
+++ b/addons/stock_fleet/models/fleet_vehicle_model.py
@@ -5,7 +5,7 @@ from odoo.tools import format_list
 
 
 class FleetVehicleModelCategory(models.Model):
-    _inherit = 'fleet.vehicle.model.category'
+    _inherit = ['fleet.vehicle.model.category']
 
     weight_capacity = fields.Float(string="Max Weight")
     weight_capacity_uom_name = fields.Char(string='Weight unit of measure label', compute='_compute_weight_capacity_uom_name')

--- a/addons/stock_fleet/models/stock_location.py
+++ b/addons/stock_fleet/models/stock_location.py
@@ -1,7 +1,7 @@
 from odoo import fields, models
 
 
-class StockPickingBatch(models.Model):
-    _inherit = 'stock.location'
+class StockLocation(models.Model):
+    _inherit = ['stock.location']
 
     is_a_dock = fields.Boolean("Is a Dock Location")

--- a/addons/stock_fleet/models/stock_picking.py
+++ b/addons/stock_fleet/models/stock_picking.py
@@ -4,7 +4,7 @@ from odoo import fields, models
 
 
 class StockPicking(models.Model):
-    _inherit = "stock.picking"
+    _inherit = ["stock.picking"]
 
     zip = fields.Char(related='partner_id.zip', string='Zip', search="_search_zip")
 

--- a/addons/stock_fleet/models/stock_picking_batch.py
+++ b/addons/stock_fleet/models/stock_picking_batch.py
@@ -4,7 +4,7 @@ from odoo import fields, models, api
 
 
 class StockPickingBatch(models.Model):
-    _inherit = 'stock.picking.batch'
+    _inherit = ['stock.picking.batch']
 
     vehicle_id = fields.Many2one('fleet.vehicle', string="Vehicle")
     vehicle_category_id = fields.Many2one(
